meebee/app/src/main/java/kr/lion/meebee/dto/response/VillageForecastResponse.kt

```diff
@@ -0,0 +1,72 @@
+package kr.lion.meebee.dto.response
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+import kr.lion.meebee.model.VillageForecastResult
+
+@JsonClass(generateAdapter = true)
+data class VillageForecastResponse(
+    @Json(name = "response")
+    val response: Response
+){
+    fun toUiModel(): VillageForecastResult {
+        // Ui에서 사용할 형태로 구현 예정
+        return VillageForecastResult()
+    }
```

Comment:
+) 그리고 이런 매핑 함수는 사실 그 대상이 되는 클래스 내부에서 가지고 있을 관심사는 아니긴해요.
아래와 같이 확장함수 형태로 클래스 외부에 써두는게 좋아보여요ㅎㅎ
```suggestion
data class VillageForecastResponse(
    @Json(name = "response")
    val response: Response
)

fun VillageForcecastResult.toResult(): VillageForecastResult {
    // DataSource 외부에서(?) 사용할 형태로 구현 예정
    return VillageForecastResult()
}
```

Reply:
사소한 차이지만 엄청 큰 차이기도 하네요 확실히 이런 의존성들에서 확장 함수를 쓰는게 좋은 방법인거 같아요😀